Source/KeyHandler.mm

```diff
@@ -1834,7 +1853,7 @@ - (void)_walk
     _latestWalk = _grid->walk();
 }
 
-- (InputStateChoosingCandidate *)_buildCandidateState:(InputStateNotEmpty *)currentState useVerticalMode:(BOOL)useVerticalMode
+- (InputStateChoosingCandidate *)_buildCandidateStateFromInputtingState:(InputStateInputting *)inputting   UseVerticalMode:(BOOL)useVerticalMode
```

Comment:
I believe it should be `useVerticalMode:` instead of `UseVerticalMode:`. Also, there are two extra spaces here.